index.d.ts

```diff
@@ -68,7 +68,8 @@ export namespace createStoreon {
     Event extends keyof (Events & StoreonEvents<State, Events>)
   > = (
     state: State extends object ? Readonly<State> : State,
-    data: (Events & StoreonEvents<State, Events>)[Event]
+    data: (Events & StoreonEvents<State, Events>)[Event],
+    store: StoreonStore
```

Comment:
I think this should be a `StoreonStore<State, Events>`.

Reply:
9b64f00

Reply:
Released in 3.1.1